Author: Thomas Mueller <thomas@chaschperli.ch>
Description: Disable "Product News" dashboard widget as it refers to
  upstream not to the debian package.

diff -Naur otrs-3.1.7.orig/Kernel/Config/Files/Framework.xml otrs-3.1.7/Kernel/Config/Files/Framework.xml
--- otrs-3.1.7.orig/Kernel/Config/Files/Framework.xml	2012-06-21 13:40:05.000000000 +0200
+++ otrs-3.1.7/Kernel/Config/Files/Framework.xml	2012-07-02 18:50:10.925635570 +0200
@@ -5198,7 +5198,7 @@
                 <Item Key="URL">http://otrs.org/product.xml</Item>
                 <Item Key="Block">ContentLarge</Item>
                 <Item Key="Group">admin</Item>
-                <Item Key="Default">1</Item>
+                <Item Key="Default">0</Item>
                 <Item Key="CacheTTLLocal">1440</Item>
             </Hash>
         </Setting>
